packages/host/tests/acceptance/code-mode-test.ts

```diff
@@ -282,12 +284,13 @@ module('Acceptance | code mode tests', function (hooks) {
       )}`,
     );
 
-    await waitFor('[data-test-file="person.gts"]');
+    await waitFor('[data-test-file="pet-person.gts"]');
 
-    await click('[data-test-file="person.gts"]');
+    await click('[data-test-file="pet-person.gts"]');
 
-    await waitFor('[data-test-file="person.gts"]');
-    assert.dom('[data-test-file="person.gts"]').hasClass('selected');
+    await waitFor('[data-test-file="pet-person.gts"]');
+    assert.dom('[data-test-file="pet-person.gts"]').hasClass('selected');
+    assert.dom('[data-test-file="person.gts"]').doesNotHaveClass('selected');
```

Comment:
This is asserting against this bug:

<img width="251" alt="Boxel 2023-09-15 13-22-59" src="https://github.com/cardstack/boxel/assets/43280/4363e554-8043-4115-b719-ad522250001a">